### PR TITLE
feat(standups): added options menu tooltip

### LINKED ISSUE #7762: Standups: more menu needs tooltip
## Issue - Enhancement

The "more menu" in standups is missing a tooltip when hovered or focused:

Captura de pantalla 2023-02-13 a la(s) 12 27 03 p m

The tooltip should look and behave the same way other buttons do:
Captura de pantalla 2023-02-13 a la(s) 12 27 51 p m

...and I suggest the tooltip says "Options".

### Acceptance Criteria (optional)
When hovered or focused, the more menu in Standups shows a tooltip that says "Options"

**Estimated effort:** 1 hr

### PR DESCRIPTION
**Pull request by BartoszJarocki**
# Description

Fixes #7762

## Demo

CleanShot 2023-02-14 at 08 26 31

## Testing scenarios

[Please list all the testing scenarios a reviewer has to check before approving the PR]

- [ ] Hover over standups options menu and check if there's a tooltip showing

## Final checklist

- [x] I checked the [code review guidelines](../docs/codeReview.md)
- [x] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [x] I have performed a self-review of my code, the same way I'd do it for any other team member
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [x] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [x] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [x] PR title is human readable and could be used in changelog

### CODE REVIEW

File: packages/client/components/TeamPrompt/TeamPromptOptions.tsx [hunk 1]
```diff
@@ -42,18 +49,24 @@ const TeamPromptOptions = (props: Props) => {
   )
 
   return (
-    <>
-      <OptionsButton ref={originRef} onClick={togglePortal}>
+    <div ref={tooltipOriginRef}>
```
Comment (BartoszJarocki) [inline packages/client/components/TeamPrompt/TeamPromptOptions.tsx:52]: another div is needed as we need 2 refs, one for menu, one for tooltip

**Comment by BartoszJarocki** [comment]
@jmtaber129 yeah same here. @mattkrick or @Dschoordsch do you have any ideas how can we do it without the second div?

**Comment by BartoszJarocki** [comment]
@jmtaber129 added `mergeRefs` function, lmk what do you think

**Review by jmtaber129 — APPROVED** [review]
`mergeRefs` + the rest LGTM, but since it's a new pattern the new util should be reviewed by maintainers

**Comment by BartoszJarocki** [comment]
thanks! and I agree. @Dschoordsch do you mind taking a look?

**Review by Dschoordsch — APPROVED** [review]

**Comment by Dschoordsch** [comment]
Works and looks good, nice work! I agree with adding the `mergeRef` function

**Pull request merged**